code/modules/hydroponics/gene_modder.dm

```diff
@@ -443,3 +443,14 @@
 	if(gene && (istype(gene, /datum/plant_gene/core/potency)))
 		. += "<span class='notice'>Percent is relative to potency, not maximum volume of the plant.</span>"
 	. += "The write-protect tab is set to [src.read_only ? "protected" : "unprotected"]."
+
+/obj/machinery/plantgenes/AltClick(mob/user)
+	if(!user.canUseTopic(src, !issilicon(user)))
+		return
+	var/obj/item/I = user.get_active_held_item()
+	eject_disk()
+	if(istype(I, /obj/item/disk/plantgene))
+		if(!user.transferItemToLoc(I, src))
+			return
+		disk = I
+		to_chat(user, "<span class='notice'>You add [disk] to the machine.</span>")
```

Comment:
Remove the part where it puts the old disk in your hand, thats what clicking is for, leave alt click to just removing the disk